chromium_src/chrome/installer/util/google_update_settings.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
this doesn't seem right, we only added GOOGLE_CHROME_BUILD for part of the file, I don't think we want it defined for the whole file

Reply:
The patch had it defined in 41 (after the headers and the other patch) and undefined in line 836 at the end of the file, as far as I can tell.